src/components/login/Login.tsx

```diff
@@ -0,0 +1,65 @@
+import { useContext, useState } from "react";
+import { AuthContext } from "../../contexts/AuthContext";
+import useLogin from "../../hooks/useLogin";
+import SignUpButton from "./SignUpButton";
+import LoginForm from "./LoginForm";
+import LoginHeader from "./LoginHeader";
+import Middel from "../ui/Middel";
+
+const Login = () => {
+  const { loginUser } = useLogin();
+
+  const { login } = useContext(AuthContext);
+
+  const [email, setEmail] = useState<string>("");
+  const [password, setPassword] = useState<string>("");
+  const [loading, setLoading] = useState<boolean>(false);
+  const [errorMessage, setErrorMessage] = useState<string | undefined>(undefined);
+
+  const handleLogin = async (event: React.FormEvent) => {
+    event.preventDefault();
+
+    setLoading(true);
+
+    const response = await loginUser(email, password);
+
+    if ("error" in response && response.error) {
```

Comment:
Hoeft `"error" in response` ? Is `response.error` niet voldoende?
Indien ja, wijzig dit dan ook bij 'src/components/register/Register.tsx'.

Reply:
Als ik dat niet doe dan klaag typescrpt omdat het zo gezegt niet bestaand, weet jij mischien een oplossing?

Reply:
Probeer eens `if (response && response.error) {...}`